ddp-workspace/projects/ddp-testboston/src/app/app-guids.ts

```diff
@@ -1,5 +1,6 @@
 export const AppGuids = {
     Consent: 'CONSENT',
     Dashboard: 'DASHBOARD',
-    Covid: 'COVID_SURVEY'
+    Covid: 'COVID_SURVEY',
```

Comment:
Could we rename this to something like ActivityGuids ? AppGuid does not sound right to me.